### PR TITLE
Feature Description Added  Importing App Manifest 

### LINKED ISSUE #7481: [Docs] Feature Description Needed: Importing Docker Compose Apps
#### Current State
Users can import their Docker Compose apps into Meshery as Meshery Apps. A description of this feature, how to use it in both Meshery clients, and a characterization of its behavior is needed.

#### Desired State
New page / section in Meshery Docs

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by thisiskaransgit**
**Notes for Reviewers**

This PR fixes #7481 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 124413e99b3bafe923b1220616099c7c7e0cdd48 at: https://6466885638306f005d594ce4--meshery-docs-preview.netlify.app

File: docs/pages/guides/Importing-Apps.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+<pre class="codeblock-pre">
+<div class="codeblock"><div class="clipboardjs">mesheryctl app import -f [file/url] -s [source-type]</div></div>
+</pre>
+
+This command enable users to import their existing applications from 
+- Helm Charts
+- k8s manifests
+- Docker Compose
```
Comment (alphaX86) [inline docs/pages/guides/Importing-Apps.md:37]: It'd be great if an example command is present

File: docs/pages/guides/Importing-Apps.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+You can import apps into Meshery using either the Meshery CLI or the Meshery UI. We will discuss both methods in detail below.
+
+
+## Using Meshery CLI:
+
+
+**Step 1: Install Meshery CLI**
+
+Before you can use the Meshery CLI to import a Docker Compose app, you must first install it. You can install Meshery CLI by following the instructions on the Meshery documentation site.
```
Comment (alphaX86) [inline docs/pages/guides/Importing-Apps.md:21]:
```suggestion
Before you can use the Meshery CLI to import a Docker Compose app, you must first install it. You can install Meshery CLI by following the instructions on the [Meshery documentation site].
```
A link would be nice

File: docs/pages/guides/Importing-Apps.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+
+
+## Using Meshery UI:
+
+
+**Step 1: Access the Meshery UI**
+
+
+To import a Docker Compose app into Meshery using the Meshery UI, you must first access the Meshery UI.
```
Comment (alphaX86) [inline docs/pages/guides/Importing-Apps.md:48]: Again, instructions to access the UI would be good

File: docs/pages/guides/Importing-Apps.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+
+To import a Docker Compose app into Meshery using the Meshery UI, you must first access the Meshery UI. 
+
+
+
+**Step 2: Navigate to the Application section in the Configuration**
+
+
+Once you have accessed the Meshery UI, navigate to the App Import page. This page can be accessed by clicking on the "Applications" menu item and then selecting "Import Application".
```
Comment (alphaX86) [inline docs/pages/guides/Importing-Apps.md:55]: A screenshot for the menu bar would be nice?

**Review by alphaX86 — CHANGES_REQUESTED** [review]
My opinions and suggestions on this PR... Thanks

**Comment by l5io** [comment]
🚀 Preview for commit 9eb358e1037343042578328b10b8e737f237356d at: https://646a61604c94eb2cc0fd3aea--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit c090addb2f87779af17e0bdc629aca6446529a9e at: https://646a65ccc369832db0029c80--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f98d30aaf2c7d70ce48bedda40feea715abc4dd1 at: https://646a6902c369832f55029d78--meshery-docs-preview.netlify.app

File: docs/pages/guides/Importing-Apps.md (hunk 1 above)
Comment (thisiskaransgit) [reply docs/pages/guides/Importing-Apps.md]: done!

File: docs/pages/guides/Importing-Apps.md (hunk 2 above)
Comment (thisiskaransgit) [reply docs/pages/guides/Importing-Apps.md]: done!

File: docs/pages/guides/Importing-Apps.md (hunk 3 above)
Comment (thisiskaransgit) [reply docs/pages/guides/Importing-Apps.md]: done!

File: docs/pages/guides/Importing-Apps.md (hunk 4 above)
Comment (thisiskaransgit) [reply docs/pages/guides/Importing-Apps.md:59]: done!

**Review by alphaX86 — APPROVED** [review]
Looks great now! LGTM

**Comment by alphaX86** [comment]
Thanks @thisiskaransgit we're good to go

**Pull request merged**